src/library/scala/util/Try.scala

```diff
@@ -141,6 +141,61 @@ sealed abstract class Try[+T] extends Product with Serializable {
     def withFilter(q: T => Boolean): WithFilter = new WithFilter(x => p(x) && q(x))
   }
 
+  /** Tests whether this is a `Success` with a given value.
+   *
+   * This is equivalent to:
+   * {{{
+   * tryComputation match {
+   *   case Success(x) => x == value
+   *   case Failure(_) => false
+   * }
+   * }}}
+   *  @example {{{
+   *  // Returns true because Success instance contains string "something" which equals "something".
+   *  Success("something") contains "something"
+   *
+   *  // Returns false because "something" != "anything".
+   *  Success("something") contains "anything"
+   *
+   *  // Returns false when method called on Failure.
+   *  Failure(new Exception("It failed")) contains "anything"
+   *  }}}
+   *
+   *  @param value the value to test.
+   *  @return `true` if the successful computation returns a value 
+   * that is equal (as determined by `==`) to `value`, `false` otherwise.
+   */
+  def contains[U >: T](value: U): Boolean
```

Comment:
Is there an opportunity for typesafe contains? `isSuccess(T)` or `isSuccessful(T)`?